src/test_server.py

```diff
@@ -0,0 +1,46 @@
+"""Tests to make sure client and server can communicate via sockets."""
+
+
+import pytest
+
+
+def test_response_ok():
+    """Test to see if valid client request will return a 200 OK message."""
+    from client import client
+    response = ("HTTP/1.1 200 OK\r\n"
+        "Server: Teddy Bear\r\n"
+        "Content-Type: text/html\r\n"
+        "Connection: Closed\r\n\r\n"
+        "<html>\r\n"
+        "<body>\r\n"
+        "<h1>Hello, World!</h1>\r\n"
+        "</body>\r\n"
+        "</html>")
+    assert client("GET the bl®rg woot") == response
```

Comment:
Im confused about what you want for this for step 1. https://codefellows.github.io/sea-python-401d5/assignments/http_server_1.html. The assignment says to build server side functions for response_ok and response_failed. How should we determine which response to send based on the request for step 1? In step 2 we implement server handling for valid and well-formed requests (https://codefellows.github.io/sea-python-401d5/assignments/http_server_2.html), so my thinking was the requirements for the server responses in step 1 were trivial.